lib/runtime/src/protocols/annotated.rs

```diff
@@ -61,6 +61,16 @@ impl<R> Annotated<R> {
         }
     }
 
+    /// Create a new annotated stream with complete final event
+    pub fn from_complete_final() -> Self {
+        Self {
+            data: None,
+            id: None,
+            event: Some("complete_final".to_string()),
```

Comment:
EOS = end of stream?

Reply:
Yes.

Updated that the `Annotated` struct is no longer used for tracking end of stream.